src/index.tsx

```diff
@@ -766,7 +766,12 @@ const StyledEditor = styled("div")<{
       }
     }
   }
-
+  .heading-content {
+    &:before {
+      content: "​";
```

Comment:
Note: There is an invisible character in the `content` here, I'm surprised GitHub doesn't make an exception and display it given they've been known to cause problems when misplaced.